TowerDefense/TowerDefense.cs

```diff
@@ -195,7 +195,7 @@ private void HandleMouse()
             {
                 if (mouse.MouseState.LeftButton == ButtonState.Pressed)
                 {
-                    Exit();
+                    //Exit();
```

Comment:
iOS the user controls when you close the app, you never have the ability to terminate your own app